Jellyfin.Server/Program.cs

```diff
@@ -59,6 +59,18 @@ static Task ErrorParsingArguments(IEnumerable<Error> errors)
                 return Task.CompletedTask;
             }
 
+            // The dotnet swagger tool passes in an --applicationName that isn't allowed by the CLI parser
+            // Skip over it and add the "nowebclient" option to the server to allow it to start successfully
+            // Swagger is used to generate the OpenAPI document using the startup assembly and then exit the app
+            if (Assembly.GetEntryAssembly()?.GetName().Name == "dotnet-swagger")
```

Comment:
if i remember correctly, all arguments you pass to swagger that are not parsed by it, will be handed down to the application itself. So if running the app without webclient, you can just put the cmd argument for that in the swagger cmd instead

Reply:
Well the `dotnet swagger` tool doesn't actually support forwarding additional arguments to the application.
Trying to pass in and additional argument of  "--nowebclient" has the swagger command fail with exit code 1.
 ```
dotnet swagger tofile --help
Usage: dotnet swagger tofile [options] [startupassembly] [swaggerdoc]

startupassembly:
  relative path to the application's startup assembly

swaggerdoc:
  name of the swagger doc you want to retrieve, as configured in your startup class

options:
  --output:  relative path where the Swagger will be output, defaults to stdout
  --host:  a specific host to include in the Swagger output
  --basepath:  a specific basePath to include in the Swagger output
  --serializeasv2:  output Swagger in the V2 format rather than V3
  --yaml:  exports swagger in a yaml format
```